content/posts/aws-workplace.md

```diff
@@ -0,0 +1,96 @@
+---
+draft: false
+date: 2020-05-04T01:16:13+02:00
+title: "Przygotowanie środowiska do pracy z AWS"
+slug: "aws-workplace"
+series: ["aws-bootstrap"]
+tags: [aws]
+---
+
+_To pierwszy wpis z serii o fundamentach AWS. Kolejne będą pojawiały się w [tym spisie]({{< ref "/series/aws-bootstrap" >}})._
+
+Za każdym razem gdy siada się do chmurowego projektu na nowym koncie AWS, chce się wypróbować nowe lub nieznane usługi albo pouczyć się do certyfikatu, pojawia się problem przygotowania i skonfigurowania środowiska pracy.
+
+Chmura Google rozwiązuje ten problem (przynajmniej częściowo) poprzez swój [Cloud Shell](https://cloud.google.com/shell) - darmowy terminal w przeglądarce, w którym można skorzystać z zawsze aktualnych narzędzi konsolowych do pracy z GCP. W przypadku AWS nie jest już tak prosto i - choć da się stworzyć podobne narzędzie samemu (np. instancja EC2 z przypisaną odpowiednią rolą IAM) - to będzie to wymagało dodatkowej pracy, a użyte zasoby będą płatne. Dlatego do celów szkoleniowo-testowych zdecydowałem się na pracę z AWS z konsoli laptopa.
```

Comment:
Warto wspomnieć o AWS Lightsail jako free resources?

Reply:
A co można dostać za free? Z tego co widzę to lightsail nie ma nawet free tier normalnego, tylko miesiąc triala.

Reply:
Oh sorry, to tego nie widzialem. Wydawalo mi sie, ze kiedys promowali to jako free forever (pod warunkiem ze to bedzie na jakims t2.szrot-na-ktorym-ledwo-process-ssh-dziala)